src/connectors/nrk-radio.js

```diff
@@ -1,7 +1,17 @@
 'use strict';
 
-Connector.playerSelector = '.mini-player';
+const filter = new MetadataFilter({
+	artist: cleanUpArtist,
+});
 
-Connector.artistTrackSelector = '.currently-playing .episode-title';
+Connector.playerSelector = '.app';
 
-Connector.pauseButtonSelector = '.paused';
+Connector.artistTrackSelector = '[data-test="playingEpisodeDesc"]';
+
+Connector.pauseButtonSelector = '[data-test="playPauseMiniplayer"][aria-label="Pause"]';
+
+Connector.applyFilter(filter);
+
+function cleanUpArtist(artist) {
+	return artist.replace('+', '&');
```

Comment:
`String.replace` replaces a given substring one time only. So, if you have a "K-391 + Alan Walker + Julie Bergan + Seungri" string as an artist name, the output string will be "K-391 & Alan Walker + Julie Bergan + Seungri". This should be changed to keep consistent artist format.

You can use regular expressions to replace all matches.

Reply:
Thanks for explaining - I totally didn't realise that replace when passed string would only replace the first instance. Is that the same when passing regex without global flag? I guess so.

docs: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace

Reply:
Yes, it's the same for expressions w/o a global flag.

https://jsfiddle.net/zmq548ov/

Reply:
Also, I think we could include spaces to make the filter even more robust, and to avoid unwanted replaces.

Reply:
Hi! Thank you both for the information! I'll try to fix this by using the regex. :)